CS342MemberViews/Sources/CaitlinKunchur/HobbiesRow.swift

```diff
@@ -0,0 +1,89 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+
+import SwiftUI
+
+
+public struct HobbiesRow: View {
```

Comment:
Very nice job displaying all your hobbies in your view!! 👍

Please be sure that you address all the SwiftLint errors in this file before you merge the PR.